src/app/dynamic-forms/dialogs/add-rich-text-image/add-rich-text-image.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="add-rich-text-link-dialog">
+    <h1 mat-dialog-title>
+      Set your Image
+    </h1>
+  
+    <div mat-dialog-content>
+      <form>
+        <div class="full-width-element divMarginBottom relative-pos">
+          <div class="image-labels">
+            <label class="cinchy-label" for="imageHref">
```

Comment:
`Href` should be `Src`, since that is what is injected into the `img` tag. Please make that change universally for this component.